File: C++/OddEven.cpp
#include<iostream>
using namespace std;

class Test {
    int num;
public:

    void input() {
        cout << "Enter a number:";
        cin>>num;
    }

    void check() {
        if (num % 2 == 0) {
            cout << "Number is even:" << num;
        } else {
            cout << "Number is odd:" << num;
        }
    }
};

int main() {

    Test tt;

    tt.input();
    tt.check();

    return 0;
}

File: Java/OddEven.java
public class OddEven {
    public static void main(String args[]){
        int a[]={23,44,22,33,12,34,28,75};
        System.out.println("Odd Numbers:");
        for(int i=0;i<a.length;i++){
            if(a[i]%2!=0){
                System.out.println(a[i]);
            }
        }
        System.out.println("Even Numbers:");
        for(int i=0;i<a.length;i++){
            if(a[i]%2==0){
                System.out.println(a[i]);
            }
        }
    }
}
